pkgs/development/libraries/physics/geant4/default.nix

```diff
@@ -101,6 +108,8 @@ stdenv.mkDerivation rec {
   postFixup = ''
     # Don't try to export invalid environment variables.
     sed -i 's/export G4\([A-Z]*\)DATA/#export G4\1DATA/' "$out"/bin/geant4.sh
+  '' + lib.optionalString enableQT ''
+    wrapQtAppsHook
```

Comment:
I'm not sure you are supposed to need to call this manually.

Reply:
It breaks without it.  I am not sure why, but this was what was suggested on the discord.